include/ObjectIntrospection.h

```diff
@@ -248,10 +254,12 @@ int getObjectSize(const T &ObjectAddr, size_t &ObjectSize, const options &opts,
  * production system.
  */
 template <class T>
-int __attribute__((weak))
-getObjectSize(const T &ObjectAddr, size_t &ObjectSize) {
+int getObjectSize(const T &ObjectAddr, size_t &ObjectSize) {
```

Comment:
Independently from this diff, there is some inconsistency about code style, the `&` should go with the type, not with the variable. Also, arguments should be in mixedCase.

Reply:
I don't think the former is inconsistent in OI, if you look at say `src/OICodeGen.cpp` which was manually formatted the qualifiers are attached to the name instead of the type. I can change the name of the variables though, that is inconsistent.